.github/workflows/weekly-build.yaml

```diff
@@ -0,0 +1,51 @@
+name: LVMO Weekly Image Publish
+
+# Run every Monday at 23:00 UTC
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 23 * * 1"
+env:
+  GO_VERSION: "1.17"
+  IMG: "quay.io/ocs-dev/lvm-operator:weekly"
+
+jobs:
+  build-and-publish-image:
+    name: Build and publish the LVMO image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - uses: actions/checkout@v2
+
+      # 'test' is a sub-target of 'docker-build' and so cache go build & mod
```

Comment:
what's `test` over here?

Reply:
The make docker-build-combined runs a make test automatically.

Reply:
Done